### ISSUE TITLE
`{{` cannot be used in markdown files

### ISSUE THREAD

**Issue by bryangingechen** [leanprover-community/leanprover-community.github.io#168]
We're telling staticjinja that [every `.md` file is a jinja2 template](https://github.com/leanprover-community/leanprover-community.github.io/blob/1a7f9a90bfca81144ab4ffdfac9f7eebd9003487/make_site.py#L399-L421), so if one of them contains `{{` (e.g. when talking about "weakly implicit" arguments (`⦃` in unicode), jinja2 will be unable to parse the file and then complain:
```
jinja2.exceptions.TemplateSyntaxError: unexpected char 'XX' at YY
```

This came up when testing `templates/latex.md` in #165.

**Comment by robertylewis** [comment]
Are we using this feature at all? It looks like all our jinja2 templates are .html files, but I'm not sure. Also I have no idea how to disable this processing on markdown files. So this isn't a very helpful question...

**Comment by bryangingechen** [comment]
I agree that we're not actually using the markdown files as templates, and I tried to figure out how to disable the checking but it seemed like it would require a bigger refactor, so I didn't dig too hard. 

I did a search just now and we do actually have `{{` in [`ci.md`](https://github.com/leanprover-community/leanprover-community.github.io/blame/newsite/templates/ci.md#L85) inside a yaml code block: 
```yaml
          repo: ${{ github.repository }}
          access-token: ${{ secrets.GITHUB_TOKEN }}
```

and [`contribute/doc.md`](https://github.com/leanprover-community/leanprover-community.github.io/blame/newsite/templates/contribute/doc.md#L207) inside a Lean code block:
```lean
| binder_info.strict_implicit := ("{{", "}}")
```

So this may be less of a big deal than I thought. In the example that was breaking, `{{` was in [one of the LaTeX tests that wasn't in a code block](https://github.com/leanprover-community/leanprover-community.github.io/blame/default_lean_syntax_highlight/templates/latex.md#L192) and jinja complained, so I changed it to `{  {`.
